### DISCUSSION TITLE
0.40.0

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#804] [Announcements]
## What's Changed
* test python 3.10 by @rmorshea in https://github.com/idom-team/idom/pull/767
* add asgiref dep + set default timeout on page by @rmorshea in https://github.com/idom-team/idom/pull/776
* Track contexts in hooks as state by @rmorshea in https://github.com/idom-team/idom/pull/787
* use strict equality for text, numeric, and binary types by @rmorshea in https://github.com/idom-team/idom/pull/790
* make nox-session reusable workflow by @rmorshea in https://github.com/idom-team/idom/pull/791
* update error messages by @rmorshea in https://github.com/idom-team/idom/pull/792
* Bump jsdom from 16.3.0 to 16.5.0 in /src/client by @dependabot in https://github.com/idom-team/idom/pull/774
* Bump got from 11.8.3 to 11.8.5 in /src/client by @dependabot in https://github.com/idom-team/idom/pull/772
* Fix: Accidental mutation of old model causes invalid JSON Patch by @rmorshea in https://github.com/idom-team/idom/pull/802
* Use LXML for html_to_vdom by @Archmonger in https://github.com/idom-team/idom/pull/795

**Full Changelog**: https://github.com/idom-team/idom/compare/0.39.0...0.40.0

<hr /><em>This discussion was created from the release <a href='https://github.com/idom-team/idom/releases/tag/0.40.0'>0.40.0</a>.</em>

**Comment by Archmonger** [comment]
The docs for `html_to_vdom` are slightly broken due to changing the `source` parameter to `html`

**Reply by rmorshea** [reply]
Feel free to make a follow up PR or make an issue for it.
